### PR TITLE
update libraries

### PR DESCRIPTION
**Pull request by pvighi**
also removed unused library

### CODE REVIEW

File: handlers/digital-subscription-expiry/build.sbt [hunk 1]
```diff
@@ -16,9 +16,8 @@ libraryDependencies ++= Seq(
   "com.amazonaws" % "aws-lambda-java-log4j" % "1.0.0",
   "com.amazonaws" % "aws-lambda-java-core" % "1.2.0",
   "com.amazonaws" % "aws-java-sdk-s3" % "1.11.265",
-  "log4j" % "log4j" % "1.2.17",
```
Comment (pvighi) [inline handlers/digital-subscription-expiry/build.sbt]: This one doesn't seem to be needed. 
@johnduffell Is there we had it ? if not we should remove it from all lambdas in this project

Comment (johnduffell) [reply handlers/digital-subscription-expiry/build.sbt]: maybe not, I thought it was somehow needed by the lambda library

**Comment by johnduffell** [comment]
👍

**Comment by pvighi** [comment]
addresses snyk report by overriding jackson-databind, same approach as https://github.com/guardian/zuora-auto-cancel/pull/65

**Comment by johnduffell** [comment]
👍

**Pull request merged**